packages/vm/src/wasm_backend/gatekeeper.rs

```diff
@@ -637,11 +630,7 @@ impl FunctionMiddleware for FunctionGatekeeper {
             | Operator::F32x4RelaxedMin
             | Operator::F32x4RelaxedMax
             | Operator::F64x2RelaxedMin
-            | Operator::F64x2RelaxedMax
-            | Operator::F32x4Fma
-            | Operator::F32x4Fms
-            | Operator::F64x2Fma
-            | Operator::F64x2Fms => {
+            | Operator::F64x2RelaxedMax => {
```

Comment:
Wonder from where these guys come from.

Reply:
There seems to be some new Wasm extension here:

```
            // Relaxed SIMD operators
            // https://github.com/WebAssembly/relaxed-simd
```